packages/braid-design-system/src/lib/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -9,7 +9,10 @@ import { Box } from '../Box/Box';
 import * as styles from './OverflowMenu.css';
 
 export interface OverflowMenuProps
-  extends Omit<MenuRendererProps, 'trigger' | 'align' | 'offsetSpace'> {
+  extends Omit<
+    MenuRendererProps,
+    'size' | 'trigger' | 'align' | 'offsetSpace'
```

Comment:
Rather than omitting, I would suggest adding `size` support to OverflowMenu too! Shouldnt be much work across docs, screenshots, etc. Just have to pass the size to `MenuRenderer` and the icon here too.